src/base/utils/net.h

```diff
@@ -28,13 +28,26 @@
 
 #ifndef BASE_UTILS_NET_H
 #define BASE_UTILS_NET_H
+
+#include <QList>
+
+class QHostAddress;
 class QString;
+class QStringList;
+template <typename T, typename U> struct QPair;
 
 namespace Utils
 {
     namespace Net
     {
+        using Subnet = QPair<QHostAddress, int>;
+
         bool isValidIP(const QString &ip);
+        bool canParseSubnet(const QString &subnetStr);
+        bool parseSubnet(const QString &subnetStr, Subnet &subnet);
+        bool isLoopbackAddress(const QHostAddress &addr);
+        bool isIPInSubnets(const QHostAddress &addr, const QList<Subnet> &subnets);
```

Comment:
The function name is misleading. From the name it's unclear what does is test: whether the IP is in any of the subnets (which it actually does) or does it belong to all of them.

Reply:
What would you propose renaming it to?

Reply:
Maybe `isIPInRange()`?

Reply:
Updated.